src/services/montage-helper.service.js

```diff
@@ -5,7 +5,6 @@
 		.module('montage')
 		.factory('montageHelper', montageHelper);
 
-	function montageHelper(colorPrimary, colorDanger, montage, authService, ngProgressFactory) {
```

Comment:
_Learning Opportunity_:  Atomic commits should contain the _complete_ idea. Each commit should keep the code in a consistent state. That means that a commit should not represent a part of a work in progress that breaks code.

Removing this line breaks the application. This commit should include the change in commit [fdaa39bd](https://github.com/Montage-Inc/montage-frontend/pull/62/commits/fdaa39bdb09846cc27fa1ecf5290a7b4c52adc28)

```
function montageHelper(colorPrimary, montage, authService, ngProgressFactory) {
```